### DISCUSSION TITLE
No settings anymore?

### DISCUSSION THREAD

**Discussion by Bindkedon** [aclap-dev/video-downloadhelper#892] [Help and Q&A]
### Your question

The settings are really hard to find! I needed half an hour to find them... down in the sidebar - who searches here?

### Addon & CoApp version and details

version: 9.0.2.3
target: mozilla
channel: stable
lang: de
coapp: {"found":true,"path":"C:\\Program Files\\DownloadHelper CoApp\\vdhcoapp.exe","version":"2.0.19","new_version":false}
license: {"unset":true}
platform: x86-64 win
UA: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:127.0) Gecko/20100101 Firefox/127.0
{
    "used_sidebar_button": true,
    "used_popup_button": true,
    "download_directory": 
    "last_download_directory": {
        "__serializer_tag": "option_some",
        "__serializer_value": {
            "__serializer_tag": "primitive",
           
        }
    },
    "view_options": {
        "all_tabs": false,
        "low_quality": false,
        "sort_by_status": true,
        "sort_reverse": false,
        "show_button_clean": true,
        "show_button_clean_all": false,
        "show_button_convert_local": false,
        "hide_downloaded": false
    },
    "default_action": "download_as",
    "use_sidebar": true,
    "open_count_store": 11,
    "successfull_dl": 64
}

**Comment by paulrouget** [comment]
> The settings are really hard to find! I needed half an hour to find them... down in the sidebar - who searches here?

That's the first time I heard that. They are right here where all the other buttons are.

<img width="342" alt="Screenshot 2024-06-19 at 11 02 33" src="https://github.com/aclap-dev/video-downloadhelper/assets/373579/d643f017-6961-4ebc-a8da-88bbf1a2bed8">
